fs/combineFunctions.go

```diff
@@ -0,0 +1,49 @@
+package fs
```

Comment:
Rename this file to accurately reflect the contents; it is not (and should not be) combine-specific.

The functions in this file need tests.